components/script/dom/htmlscriptelement.rs

```diff
@@ -913,18 +919,17 @@ impl HTMLScriptElement {
 
             let text_rc = Rc::new(text);
 
-            // TODO: Fix step number or match spec text. Is this step 32.1?
-            let result = Ok(ScriptOrigin::internal(
-                Rc::clone(&text_rc),
-                base_url.clone(),
-                options.clone(),
-                script_type,
-                self.global().unminified_js_dir(),
-            ));
-
-            // TODO: Fix step number or match spec text. Is this step 32.2?
+            // Step 32.2: Switch on el's type:
             match script_type {
                 ScriptType::Classic => {
+                    let result = Ok(ScriptOrigin::internal(
+                        Rc::clone(&text_rc),
+                        base_url.clone(),
+                        options.clone(),
+                        script_type,
+                        self.global().unminified_js_dir(),
+                    ));
```

Comment:
I think the TODO comments should be kept? It's still unclear where this block is specified.